evm/src/cpu/kernel/opcodes.rs

```diff
@@ -20,6 +20,9 @@ pub(crate) fn get_opcode(mnemonic: &str) -> u8 {
         "MULMOD" => 0x09,
         "EXP" => 0x0a,
         "SIGNEXTEND" => 0x0b,
+        "ADDFP254" => 0x0c,
+        "MULFP254" => 0x0d,
+        "SUBFP254" => 0x0e,
```

Comment:
just out of curiosity is there a systematic reason for how these are enumerated (e.g. why had these numbers been skipped)?

Reply:
We have to support all the [EVM opcodes](https://ethereum.org/en/developers/docs/evm/opcodes/) but can fill the gaps with whatever additional operations we want :)